services/github/src/handlers.js

```diff
@@ -504,17 +490,56 @@ async function statusHandler(message) {
     `Attempting to update status of the checkrun for ${organization}/${repository}@${sha} (${taskState.conclusion})`,
   );
   try {
+    const taskDefinition = await this.queueClient.task(taskId);
+    debug(`Result status. Got task build from DB and task definition for ${taskId} from Queue service`);
+    const {customCheckRun} = (taskDefinition.extra && taskDefinition.extra.github) || {};
+
+    let customCheckRunText = '';
+    if (customCheckRun && customCheckRun.textArtifactName) {
+      const scopedQueueClient = this.queueClient.use({
+        authorizedScopes: taskDefinition.scopes,
+        credentials: this.context.cfg.taskcluster.credentials,
+      });
+
+      const url = this.queueClient
+        .buildUrl(scopedQueueClient.getArtifact, taskId, runId, customCheckRun.textArtifactName);
+
+      let res = await utils.throttleRequest({url, method: 'GET'});
+
+      if (res.status >= 400) {
+        /*
+          Some possible errors:
+            - insufficient scopes: statusCode 403
+            - no artifact entity in DB, no artifact in S3: statusCode 404
+        */
+        await this.createExceptionComment({
+          debug,
+          instGithub,
+          organization,
+          repository,
+          sha,
+          error: res.response.error,
+        });
+        await this.monitor.reportError(res.response.error);
```

Comment:
If this is a 400, we don't need to log it as an error here. The service is operating correctly and the user has made a mistake.